frontend/src/components/PlanHome/PlanHome.tsx

```diff
@@ -0,0 +1,152 @@
+import styles from './PlanHome.module.css'
+import editButton from '../../assets/editButton.png'
+import addMemberButton from '../../assets/addMemberButton.png'
+import { useParams } from 'react-router-dom'
+import { fetchPlanData } from '../Plan/hook/fetchPlanData'
+import { loadingStatus } from '../../utils/loadingStatus'
+import { fakeUserData } from '../PlanForm'
+import { MemberList } from './MemberList'
+import { PlanStage } from '../../utils/types'
+
+import { buildStyles, CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+
+type planTaskCompleted = number;
+type planPercentage = number;
+
+const planProgressCalculator = (currentStage: PlanStage): Array<planPercentage & planTaskCompleted> => {
```

Comment:
`planPercentage & planTaskCompleted` doesn't really make sense as a typing cuz this is just `number & number` which is just `number`